.github/workflows/sauce.yml

```
@@ -0,0 +1,25 @@
+name: Sauce Labs
+on: push
+concurrency: sauce-labs
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
```

```suggestion
        uses: actions/checkout@v2
```

The action we copied from was a bit outdated

---

Changed in b660785